dev/com.ibm.ws.security.authentication.tai/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -55,3 +55,7 @@ properties.desc=Collection of properties for the interceptor.
 
 extraProperties.name=Interceptor properties
 extraProperties.description=Collection of properties for the interceptor.
+
+disableLtpaCookie=Disable LTPA cookie
+disableLtpaCookie.desc=Do not create an LTPA cookie for trust association interceptor (TAI). 
```

Comment:
**change**
Do not create an LTPA cookie for trust association interceptor (TAI).
**to**
Do not create an LTPA cookie for the trust association interceptor (TAI).